### ISSUE TITLE
Video thumbnail in Android (with autoplay set to false)

### ISSUE THREAD

**Issue by alexiscoelho** [nstudio/nativescript-videoplayer#157]
I'm having an issue that only occurs in Android.
When loading a video with the autoplay option turned off everything thing works perfectly in iOS, a thumbnail is displayed and on top of it the play button is displayed as well, but on Android I just get a white area with no thumbnail or anything else. I need to tap on that area and then on Android I can see the controls and start playing the video.
Is this the expected behaviour or I have something wrong?

**Comment by edusperoni** [comment]
You can work around this by seeking to a frame (https://stackoverflow.com/questions/17079593/how-to-set-the-preview-image-in-videoview-before-playing). Unfortunately, the android mediaplayer does not show thumbs when the video is loaded.

There are a few alternatives worth considering, like overlaying a thumbnail image on top of the player.

**Comment by alexiscoelho** [comment]
Yes, I thought about that. And overlaying an image/thumbnail, play button and so on it's what I'm implementing at the moment.
But really nice would be an out of the box feature. The alternative I'm implementing requires more work on the backend, cms and content manage to add that thumbnail data that would not be necessary if this feature worked on android as it works so nice on iOS.
